Jenkinsfile.update-grobid-and-trainer

```diff
@@ -0,0 +1,17 @@
+def grobidTag = params.grobid_tag
```

Comment:
wasn't sure whether the parameter name should be `grobid_tag` or `grobidTag`

Reply:
For some reason `grobid_tag` feels more idiomatic, but the variable is definitely camel case, `def grobidTag`